tutorials/data-lake-sof-multiple-directories/data-lake-sof-multiple-directories.md

```diff
@@ -0,0 +1,238 @@
+---
+title: Use SQL on Files to query data from multiple directory
+description: It will teach you how to setup SQL on Files, optimize SQL on Files performance using a folder and subfolder directory.
+auto_validation: true
+time: 45
+tags: [ tutorial>intermediate, software-product>sap-hana]
+primary_tag: software-product-function>sap-hana-cloud\,-data-lake
+author_name: Ronit Nagwekar
+author_profile: https://github.com/rnagweka
+parser: v2
+---
+
+## Prerequisites
+-	Have access to a licensed managed or standalone SAP HANA data lake.
+-	Installation of the HDLFSCLI. See tutorial Getting Started with Data Lake Files HDLFSCLI | Tutorials for SAP Developers.
+-	Load some structured data files inside of a SAP HANA data lake File Container.
+-	Already set up HDLFS Connection in Database Explorer - Setting Up HDLFS Connection In Database Explorer.
+
+
+## You will learn
+- How to define the database objects required to query structured data files.
+- How to add a data source for a folder and subfolder directory and query the output of that file.
+
+
+## Intro
+
+Querying structured data files (CSV, ORC, Parquet) in a HANA Data Lake file container can be done using SQL on Files. Below you will find all of the steps required to start using SQL on Files.
+If you have not yet provisioned an SAP HANA data lake, here is a great tutorial on how to do so!
+Please go through the entire tutorial on **Use SQL on Files to query data from a single file/directory**
+
+We will be using the Orders table as a reference.
+
+---
+
+### STEP 1: Create a SQL on Files remote table in Files service
+
+Note that the table definition will need to match the expected schema/data types of the columns in the structured file you are trying to query. Login to the SQL console as User1 or HDLADMIN and execute all the queries associated with this tutorial. I will be executing the queries through HDLADMIN.
+
+Firstly, we create a SCHEMA in files service.
```

Comment:
Please don't use the "royal we" when addressing the user. Use "you" instead.
Please check the [style guide](https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9#mcetoc_1dtvf24ta2) for details.